scripts/maps/uniRescueMap.json

```diff
@@ -1,5 +1,16 @@
 {
-  "0xFBb1b73C4f0BDa4f67dcA266ce6Ef42f520fBB98": "10915644380000000000",
-  "0x64B6eA66877a64c2AaB5a5b446e475e4Aa73df2c": "51032341710000000000",
-  "0x2FAF487A4414Fe77e2327F0bf4AE2a264a776AD2": "49000000000000000000"
-}
+  "0xFBb1b73C4f0BDa4f67dcA266ce6Ef42f520fBB98": {
```

Comment:
overall seems to work but script did never finish for aave for me

Reply:
for aave it takes about 40 min aprox